source/infrastructure/secrets.html.md.erb

```diff
@@ -98,16 +98,15 @@ Raise a PR in the `govwifi-build` repo on Github. Ask another team member to tes
 
 New members must import all the existing gpg keys that are stored.
 
-To do this read in all the keys and import them:
+To do this read in all the keys in `passwords/.gpg-id` and import them:
 
 ```
-all_key_ids=`cat passwords/.gpg-id`
-for key_id in $all_key_ids; do gpg --keyserver keyserver.ubuntu.com --receive-keys "$key_id"; echo "${key_id}:6" | gpg --import-ownertrust; done
+$ gpg --keyserver keyserver.ubuntu.com --receive-keys <key_id> && gpg --import-ownertrust
```

Comment:
Does having the loop in the doc make it easier to copy and paste for the reader?

Reply:
So the loop doesn't work, I tried fixing it but it became a rabbit hole. For now, I think it's sensible to remove the broken loop and then try to improve it when it's next time to review the doc.

I discovered it was broken by doing exactly that: copying and pasting into the terminal and then spending about 20min trying to figure out what was going on.